authentication/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.auth.models import User
+# Create your models here.
+
+class UserProfile(models.Model):
+  uid = models.CharField(max_length=64, primary_key=True, editable=False)
```

Comment:
Uid should be stored as the username, right? So we don't need it.

Reply:
If we remove uid, it would be required to add id field and populate it. Also as you have mentioned to add handle field, so what's the need to have username field? Shouldn't we keep it blank?